python/setup.py

```diff
@@ -203,11 +203,9 @@ def get_option_from_sys_argv(option_str):
 
     v, _, _ = platform.mac_ver()
     if v:
-      v = float('.'.join(v.split('.')[:2]))
-      if v >= 10.12:
-        extra_compile_args.append('-std=c++11')
+      extra_compile_args.append('-std=c++11')
```

Comment:
Why does this need to be conditional on `v`, or on macOS for that matter? If you now need C++11, don't you *always* (or at least on all non-Windows systems; Windows might be different) want to use `-std=c++11`?